packages/app/src/ui/pages/Login/useLogin.ts

```diff
@@ -43,9 +43,13 @@ export const useLogin = (): IUseLoginData => {
       return;
     }
 
-    if (!isKeepOpen) {
-      dispatch(closePopup());
-    }
+    // TODO: I am not sure yet why is that is the cause for that bug:
+    // (https://github.com/CryptKeeperZK/crypt-keeper-extension/issues/992)
+    // When commented the bug was resolved.
+    // I think because at that scenario described in the bug issue, the pendingRequests would be already zero.
+    // if (!isKeepOpen) {
+    //   dispatch(closePopup());
+    // }
```

Comment:
It should be here. If you remove it, modal will not close at all if there is only one login step.